### PR TITLE
Update device_tracker.owntracks_http.markdown

### PR DESCRIPTION
**Pull request by flavio20002**
Track battery with a template sensor.

### CODE REVIEW

File: source/_components/device_tracker.owntracks_http.markdown [hunk 1]
```diff
@@ -34,3 +34,17 @@ Open OwnTracks and go to Connection preferences:
...
  
 Host example: If I host my Home Assistant at `https://example.duckdns.org`, my name is Paulus and my phone is a Pixel I would set the host to be `https://example.duckdns.org/api/owntracks/paulus/pixel`. This will result in an entity with an ID of `device_tracker.paulus_pixel`. You can pick any name for the user and the device.
+
+You can track the battery of you device, by defining a template sensor:
+
+```yaml
+# Example configuration.yaml entry for battery monitoring
+device_tracker:
+    - platform: template
+    sensors:
```
Comment (DubhAd) [inline source/_components/device_tracker.owntracks_http.markdown:44]: The indenting here is off - platform should have four less spaces before it.

File: source/_components/device_tracker.owntracks_http.markdown [hunk 2]
```diff
@@ -34,3 +34,17 @@ Open OwnTracks and go to Connection preferences:
...
+      pixel_battery:
+        value_template: '{{ states.device_tracker.paulus_pixel.attributes.battery|int }}'
```
Comment (DubhAd) [inline source/_components/device_tracker.owntracks_http.markdown:46]: You should consider using a default for when the battery level isn't defined:
`'{{ states.device_tracker.paulus_pixel.attributes.battery|default(0)|int }}'``

File: source/_components/device_tracker.owntracks_http.markdown [hunk 3]
```diff
@@ -34,3 +34,17 @@ Open OwnTracks and go to Connection preferences:
...
+        unit_of_measurement: "%"
+        entity_id: 
+        - device_tracker.paulus_pixel
+```
```
Comment (DubhAd) [inline source/_components/device_tracker.owntracks_http.markdown:50]: I'd suggest (and it is just a suggestion) adding an icon template to make a pretty icon that reflects the battery level:

```      icon_template: >
        {% set battery_level = states.device_tracker.person1_tablet.attributes.battery|default(0)|int %}
        {% set battery_round = (battery_level / 10) |int * 10 %}
        {% if battery_round >= 100 %}
          mdi:battery
        {% elif battery_round > 0 %}
          mdi:battery-{{ battery_round }}
        {% else %}
          mdi:battery-alert
        {% endif %}```

(that one came from work done by @skalavala )

That'll then match up with battery level reports by the various components that support it.

File: source/_components/device_tracker.owntracks_http.markdown (hunk 2 above)
Comment (fabaff) [inline source/_components/device_tracker.owntracks_http.markdown:46]: Please check https://www.home-assistant.io/developers/documentation/standards/#templates

**Pull request closed without merging**